AddShippingZone.tsx

```diff
@@ -1,18 +1,138 @@
-import React from "react";
-import { Button, SafeAreaView, Text, View } from "react-native";
+import React, { useState } from "react";
+import {
+  Linking,
+  Platform,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from "react-native";
 import { useNavigation } from "@react-navigation/native";
+import CheckBox from "@react-native-community/checkbox";
+import { ToolbarActionButton } from "./ToolbarActionButton";
 
 const AddShippingZone = () => {
   const navigation = useNavigation();
 
+  const [isLimitEnabled, setLimitEnabled] = useState(false);
+
+  React.useEffect(() => {
+    navigation.setOptions({
+      headerRight: () =>
+        ToolbarActionButton({
+          label: "Save",
+          onPress: () => console.log("Toolbar action button pressed"),
+        }),
+    });
+  }, [navigation]);
+
+  const _renderPostCodes = function () {
+    if (isLimitEnabled) {
+      return (
+        <View style={{ marginTop: 10 }}>
+          <Text style={styles.labelText}>Postcodes</Text>
+          <TextInput
+            selectionColor={"black"}
+            style={[styles.textInput, { marginTop: 10 }]}
+            multiline={true}
+            placeholder="List 1 postcode per line"
+          />
+          <Text
+            style={{
+              marginTop: 10,
+              color: "gray",
+            }}
+          >
+            Postcodes containing wildcards (e.g. CB23*) or fully numeric ranges
+            (e.g. 90210...99000) are also supported. Please see the shipping
+            zones{" "}
```

Comment:
I wonder if we should move all these texts to a separate file, to be prepared when we need to deal with localization 🤔

Reply:
Yeah, I agree. Let's handle it in #41